english/fork-jam/choco-chalice/choco-chalice.md

```diff
@@ -7,3 +7,4 @@ For some reason, you cannot get the thought of smores out of your head.
 Do you:
 
 [Start screaming.](start-screaming/start-screaming.md)
+[Search for the perfect marshmallow stick.](find-stick/find-stick.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.